pathd fails to build on aarch64-linux

<!--

*** ATTENTION ***

YOU MUST READ THIS TO HAVE YOUR ISSUE ADDRESSED

PLEASE READ AND FILL OUT THIS TEMPLATE

NEGLECTING TO PROVIDE INFORMATION REQUESTED HERE WILL RESULT IN
SIGNIFICANT DELAYS ADDRESSING YOUR ISSUE

ALWAYS PROVIDE:
- FRR VERSION
- OPERATING SYSTEM VERSION
- KERNEL VERSION

FAILURE TO PROVIDE THIS MAY RESULT IN YOUR ISSUE BEING IGNORED

FOLLOW THESE GUIDELINES:

- When reporting a crash, provide a backtrace
- When pasting configs, logs, shell output, backtraces, and other large chunks
  of text, surround this text with triple backtics

  ```
  like this
  ```

- Include the FRR version; if you built from Git, please provide the commit
  hash
- Write your issue in English

-->

---------------

**Describe the bug**
Building frr 8.1 on aarch64-linux errors out during pathd build in clippy:

```
while processing pathd/pathd:
Traceback (most recent call last):
  File "./python/xrelfo.py", line 377, in _main
    xrelfo.load_file(fn)
  File "./python/xrelfo.py", line 282, in load_file
    self.load_elf(filename, orig_filename)
  File "./python/xrelfo.py", line 324, in load_elf
    if ptr.xref is None:
  File "/build/source/python/clippy/elf.py", line 351, in __getattr__
    self._fdata[attrname] = self._fdata[attrname]()
  File "/build/source/python/clippy/elf.py", line 229, in __call__
    return self.cls(self.ptr)
  File "./python/xrelfo.py", line 70, in __init__
    super().__init__(*args, **kwargs)
  File "/build/source/python/clippy/elf.py", line 314, in __init__
    unpacked = struct.unpack(self._elfsect.endian + pspec, data)
struct.error: unpack requires a buffer of 32 bytes
make[1]: *** [Makefile:15417: pathd/pathd.xref] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/build/source'
make: *** [Makefile:6160: all] Error 2
```

<!--
A clear and concise description of what the bug is.

Put "x" in "[ ]" if you already tried following:
-->

- [x] Did you check if this is a duplicate issue?
- [ ] Did you test it on the latest FRRouting/frr master branch? On 8.1 only


**To Reproduce**
<!--
Describe the steps to reproduce the behavior.
Be as descriptive as possible.

For example:

1. Build ffr 8.1 on aarch64-linux
2. Wait for the build to error out

-->

**Expected behavior**
<!--
Write here a clear and concise description of what you expected to happen when
using the reproduction steps you provided above
-->
It should be clarified whether frr is supposed to work on aarch64.

**Screenshots**
<!--
If applicable, add screenshots to help explain your problem.
-->
n/a

**Versions**
<!--
Include your operating system type and version here

FAILURE TO PROVIDE THIS MAY RESULT IN YOUR ISSUE BEING IGNORED
-->
<!-- e.g. Fedora 24, Debian 10] -->
 - OS Version: NixOS 22.05
<!-- [e.g. Linux 5.4, OpenBSD 6.6] -->
 - Kernel: 5.15.7
<!-- e.g. 6.0, 7.4 -->
 - FRR Version: 8.1

**Additional context**
<!--
Add any other context about the problem here.
-->

In the process of packaging frr into NixOS, the x86_64-linux build and test suite work.
https://github.com/NixOS/nixpkgs/pull/151946

--- a/lib/elf_py.c
+++ b/lib/elf_py.c
@@ -636,6 +636,9 @@ static Elf_Scn *elf_find_addr(struct elffile *ef, uint64_t addr, size_t *idx)
 		Elf_Scn *scn = elf_getscn(ef->elf, i);
 		GElf_Shdr _shdr, *shdr = gelf_getshdr(scn, &_shdr);
 
+		/* virtual address is kinda meaningless for TLS sections */
+		if (shdr->sh_flags & SHF_TLS)
+			continue;
 		if (addr < shdr->sh_addr ||
 		    addr >= shdr->sh_addr + shdr->sh_size)
 			continue;

